bootstrap-joining-demo/dns-api-mesos/src/main/resources/application.conf

```diff
@@ -0,0 +1,70 @@
+akka {
+
+  loglevel = INFO
+
+  discovery.method = akka-dns
+
+  actor {
+    provider = "cluster"
+  }
+
+  remote {
+    netty.tcp {
+      hostname = ${HOST}
+      port = 2551
+      port = ${?PORT_2551}
+
+      bind-hostname = 0.0.0.0
+      bind-port = 2551
+    }
+  }
+
+  management {
+    http {
+      # The hostname where the HTTP Server for Http Cluster Management will be started.
+      # This defines the interface to use.
+      # InetAddress.getLocalHost.getHostAddress is used not overriden or empty
+      hostname = ""
+
+      port = 19999
+      port = ${?PORT_19999}
+    }
+
+    cluster.bootstrap {
+
+      # Configuration for the first phase of bootstraping, during which contact points are discovered
+      # using the configured service discovery mechanism (e.g. DNS records).
+      contact-point-discovery {
+
+        # Define this name to be looked up in service discovery for "neighboring" nodes
+        # If undefined, the name will be extracted from the ActorSystem name
+        service-name = ${MARATHON_APP_ID}
+
+        # Added as suffix to the service-name to build the effective-service name used in the contact-point service lookups
+        # If undefined, nothing will be appended to the service-name.
+        #
+        # Examples, set this to:
+        # "default.svc.cluster.local" or "my-namespace.svc.cluster.local" for kubernetes clusters.
+        service-namespace = ".marathon.mesos"
+
+        # The effective service name is the exact string that will be used to perform service discovery.
+        #
+        # Set this value to a specific string to override the default behaviour of building the effective name by
+        # concatenating the `service-name` with the optional `service-namespace` (e.g. "name.default").
+        effective-name= ${?SERVICE_NAME}
+      }
+
+      # Configured how we communicate with the contact point once it is discovered
+      contact-point {
+        # If no port is discovered along with the host/ip of a contact point this port will be used as fallback
+        fallback-port = 19999
+        fallback-port = ${?PORT_19999}
+
+        no-seeds-stable-margin = 5 seconds
```

Comment:
`no-seeds-stable-margin = 5 seconds` is a default so it can be omitted if you'd like to make the application.conf more lean ?